### ISSUE TITLE
Add option to customize css/js without creating a new template

### ISSUE THREAD

**Issue by tg44** [lightbend/paradox#293]
Hy there!

I wanted to create a new plugin which can draw flow/sequence/mermaid/railroad/etc graphs from `@flow-diagram` kind of directives.

The main problem that I can't easily inject new script headers to the template. As I played with it I can easily render the html, but for a working implementation I need to "inline script" all the sources whenever I use a diagram, and I can't inject them once.

Two possible solution came to my mind:
 - add a simple method to insert new script sources from sbt or separate file (so I can document in the install process that you need to add these things)
 - add a simple method to query and append the already inserted script tags so when we writes a new directive we have the option to add our own tags without duplication

I'm a <3h paradox user so maybe there is a valid workaround, but I didn't find it yet. (Diagrams are must-have for our documentation so I deep-dived into it.)

Also I would solve #223 with this. Also related to #37.

**Comment by ennru** [comment]
You are right AFAIK there is no way to inject things to templates from a Paradox plugin. It has to be in the selected template or the project specific overrides.
